client/settings/fraud-protection/style.scss

```diff
@@ -0,0 +1,89 @@
+.fraud-protection {
+	&-single-radio-wrapper {
+		display: flex;
+		align-items: center;
+		height: 20px;
+		margin-bottom: 8px;
+		&__item {
+			margin-right: 5px;
+			&--recommended {
+				margin-right: 5px;
+				color: #757575;
+			}
+		}
+	}
+
+	input[type='radio'] {
+		margin: 0 10px 0 0;
+		width: 20px;
+		height: 20px;
+		border-color: #757575;
+		&:checked {
+			background-color: #fff;
+			border-color: #757575;
+			&::before {
+				content: '';
+				border-radius: 50%;
+				width: 12px;
+				height: 12px;
+				margin: 3px;
+				background-color: #3582c4;
+				line-height: 1.14285714;
+			}
+		}
+	}
+
+	&__text--help-text {
+		font-size: 12px;
+		color: #757575;
+		margin: 0 0 0 30px;
+		margin-bottom: 1em;
+	}
+}
+
+.components-modal__header {
+	border-bottom: 1px solid #ddd;
+}
+
+.components-modal__body--fraud-protection {
+	padding-top: 24px;
+
+	ul {
+		list-style-type: disc;
+		margin-left: 24px;
+	}
+}
+
+.component-modal__text {
+	&--blocked {
+		color: #b32d2e;
+	}
+	&--review {
+		color: #996800;
+	}
+}
+
+.component-modal__button {
+	&--confirm {
+		float: right;
```

Comment:
The confirmation button is slightly off the modal's spacings due to the `float: right;`.

<img width="573" alt="image" src="https://user-images.githubusercontent.com/16882226/220922947-f4405128-fc31-4e01-b3b9-a3924142ca84.png">

I'd discourage the usage of `float` as it usually causes more damage than good 😄 

As a fix, we could try a different approach, such as using `margin-left: auto;` to align the button to the right:

```scss
.component-modal__button {
	&--confirm {
            display: block;
            margin-left: auto;
        }
}
```

The result would be:

<img width="573" alt="image" src="https://user-images.githubusercontent.com/16882226/220923074-6acc485b-e63d-485f-9313-388932a6b7f8.png">

Reply:
Thanks for catching this and for the advice on `float`. I've updated this in [e9f1f32](https://github.com/Automattic/woocommerce-payments/pull/5450/commits/e9f1f32f018306217e961086c1117ac87403aa66).